clang/test/Analysis/stack-addr-ps.cpp

```diff
@@ -846,3 +851,25 @@ void top(char **p) {
   foo(); // no-warning FIXME: p binding is reclaimed before the function end
 }
 } // namespace early_reclaim_dead_limitation
+
+using size_t = decltype(sizeof(int));
+void * malloc(size_t size);
+void free(void*);
```

Comment:
We usually put these at the top of the tests.
You could also consider including some system header simulator to get these declarations.

Reply:
Moved 9d0ab4e